Makefile

```diff
@@ -12,12 +12,16 @@ clean:
 	rm -rf rpc-endpoint build/
 
 test:
-
 	go test ./...
-test_up:
+migrate_up:
+	docker run -v ${CURDIR}/sql:/migrations --network host migrate/migrate -path=/migrations/ -database 'postgres://postgres:postgres@localhost:5432/test?sslmode=disable&search_path=requests' up
+migrate_down:
+	docker run -v ${CURDIR}/sql:/migrations --network host migrate/migrate -path=/migrations/ -database 'postgres://postgres:postgres@localhost:5432/test?sslmode=disable&search_path=requests' down -all
```

Comment:
very useful for migration - https://github.com/golang-migrate/migrate
it also support different db as well as redshift support.
https://github.com/golang-migrate/migrate/tree/master/database/redshift